docs/hub/_toctree.yml

```diff
@@ -280,6 +280,8 @@
     sections:
     - local: security-tokens
       title: User Access Tokens
+    - local: security-two-factor-authentication
+      title: Two-Factor Authentication
```

Comment:
maybe Multi-Factor Authentication?

Reply:
We currently name it two-factor everywhere in the hub, let's use one naming to avoid any confusion

Reply:
ok but same question for the hub then.

@Pierrci @SBrandeis @gary149 wdyt? 2FA or MFA?

Reply:
I see more often "two-factor authentication" than "multi-factor authentication"

Reply:
did an [internal poll here](https://huggingface.slack.com/archives/C01BWJU0YKW/p1707228792102379) for completeness

Reply:
the entry should also be listed on the `security` main page directly, like the others